intervention_compassion/mappings/global_intervention_mapping.py

```diff
@@ -55,7 +55,7 @@ class GlobalInterventionMapping(OnrampMapping):
                              'compassion.global.partner'),
         "RemainingAmountToRaise": "remaining_amount_to_raise",
         "FundingStatus": "funding_status",
-        "PlannedSponsorshipBeneficiariesNumber":
+        "EstimatedImpactedBeneficiaryQuantity":
```

Comment:
Remove this change, this is not good. You are in global_intervention_mapping which is not the same as intervention_mapping. In this message the fields are different. In fact I verified and the field is correctly mapped, it's just that GMC sends null values as soon as the intervention is commited.